common-npm-packages/artifacts-common/package.json

```diff
@@ -23,6 +23,6 @@
     },
     "devDependencies": {
         "@types/node-fetch": "^2.6.11",
-        "typescript": "4.0.2"
```

Comment:
We might need to ask the pipeline's team if upgrading typescript will break node 16 tasks? and if so, how does that work with buildgen and node 20 FF.

Reply:
The suggestion here was to set the version to match what's being used by the node20 task versions. Node16 versions 'shouldnt' break